guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+
+Transactions, their signatures, and addresses after they are mined remain public on the ledger forever. However, there is no personally identifiable information attached to transactions on the blockchain. The transparent property of the blockchain allows anyone to easily verify the amount of bitcoin in existence which means it also makes looking up any address or transaction trivial. This can be demonstrated going back to the first block on [January 3, 2009](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+
+> Each Bitcoin transaction contains at least one input and at least one output. This means that once a single address is known, there is a trail to follow the bitcoin.
+> 
+> <cite>As documented by <a href="https://docs.wasabiwallet.io/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-is-bitcoin-bad-in-terms-of-privacy">Wasabi Wallet</a></cite>
+
+The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know they mined the first block, which contained just one transaction, with the block reward getting sent to their address. This illustrates the possible permanence of association between an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. While it's possible break asumptions of ownership going forward, the challenge is to recover privacy once an association is made public.
+
+<!-- TODO: add graphic and heading that demonstrate points of compromise when transacting with bitcoin -->
+
+## Methods to preserve privacy
+
+Keeping private takes constant and diligent work but is not impossible to ensure an acceptable level of privacy[^1]. Let’s explore some methods and practices that help preserve privacy on a transaction level.
+
+<!-- talk about the problem as you are talking about the solution -->
+
+### Generate a new address for each payment
+
+Creating new online personas is not the only way to preserve privacy. A new address can be generated by the wallet application used to receive bitcoin. This is achieved using [HD Wallets]({{ "/guide/glossary/#hd-wallet" | relative_url }}), a standard for how private keys get generated in modern bitcoin applications. This allows each transaction not to be tied to a single address, making it difficult to associate with to the owner.
+
+> If thieves can see your income, holdings, and spending, they can use this information to target and exploit you.[^2]
+
+Using the same address multiple times does not only degrade the privacy of just one transacting party. Both the [sender](/guide/payments/receive) and [receiver](/guide/payments/receive) are put at risk. On the receiver’s side, by using a single address, senders or anyone with whom the address is shared can see the previous activity and the amount of bitcoin controlled by the owner.
```

Comment:
```suggestion
Using the same address multiple times does not only degrade the privacy of just one transacting party. Both the [sender](/guide/payments/send/) and [receiver](/guide/payments/receive) are put at risk. Re-using an address on the receivers side means anyone with whom that address is shared can see previous payments and the amount of bitcoin controlled by that address. 
```

You mention both sender and receiver are put at risk but only mention a risk related to the receiver, not sender.